experiments/make_flow/make_realnvp_normflow.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import normflows as nf
+import larsflow as lf
```

Comment:
Should this be added to the requirements?

Reply:
Done.